fail.json

```diff
@@ -11,5 +11,45 @@
                 "max_version" : 771
             }
         }
+        {
+            "name" : "Connect-Compatible_Ciphers_2",
```

Comment:
A comment to each case why they are expected to fail would be good

Reply:
Interop can't handle tests that are expected to fail, yest. I will simply remove these for now.